File: content/1-basics/3-conditionals/ts/conditionals.ts
import * as readline from 'readline';

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter a number: ', (input) => {
    const number = parseInt(input, 10);

    if (number > 0) {
        console.log('The number is positive.');
    } else if (number < 0) {
        console.log('The number is negative.');
    } else {
        console.log('The number is zero.');
    }

    rl.close();
});

File: content/1-basics/3-conditionals/cpp/conditionals.cpp
#include <iostream>

int main()
{
    int number;

    std::cout << "Enter a number: ";
    std::cin >> number;

    if (number > 0)
    {
        std::cout << "The number is positive." << std::endl;
    }
    else if (number < 0)
    {
        std::cout << "The number is negative." << std::endl;
    }
    else
    {
        std::cout << "The number is zero." << std::endl;
    }

    return 0;
}